fog/api/src/conversions.rs

```diff
@@ -38,6 +40,12 @@ impl From<&common::BlockRange> for fog_common::BlockRange {
     }
 }
 
+impl From<fog_common::BlockRange> for common::BlockRange {
+    fn from(proto_block_range: fog_common::BlockRange) -> common::BlockRange {
```

Comment:
I don't think so- I think conversion methods probably shouldn't be enforcing stuff like field correctness. If we want to enforce this, it should probably be done in the constructor.

Reply:
So then it would still be a `TryFrom` here just that this function wouldn't see the logic and it would fail calling `BlockRange::new()`
I think that makes sense, as really any malformed block should error out sooner than later.

Reply:
SG.